sharded_queue/api.lua

```diff
@@ -319,24 +293,30 @@ function sharded_queue.statistics(tube_name)
             leader_only = true
         })
 
-    local ok, stats_collection = map('tube_statistic',
-        { tube_name = tube_name }, storages)
+    local stats_collection, err = cartridge_pool.map_call('tube_statistic',
+        {{ tube_name = tube_name }}, {uri_list=storages})
 
-    if not ok or stats_collection == nil then
+    if err then
+        error(err)
+    end
+
+    if fun.iter(stats_collection):length() == 0 then
```

Comment:
If you want to check that length is greater than 0, you can simply use `next`

Reply:
Fixed https://github.com/tarantool/sharded-queue/pull/19/commits/504481ee3fe0b4af0f26484a08aeda618ce49eca